lgn-provers/src/provers/v0/query/mod.rs

```diff
@@ -11,6 +11,7 @@ pub fn create_prover(
     url: &str,
     dir: &str,
     file: &str,
+    checksum: &str,
```

Comment:
same, can't we do it simply over the whole parameter directory and be done with it ? The individual provers shouldn't have to know about this, they should expect to already get the right version.